samples/project-anfield/import_cluster_config.py

```diff
@@ -141,6 +150,22 @@ def create_vaults():
         if vault.name in available_vaults_dict.keys():
             imported_res_dict["External Targets"].append(vault.name)
             continue
+        if vault.config.qstar: # Qstar target.
+            try:
+                body = Vault()
```

Comment:
Did you test the qstar being part of a policy and export/import the policy after the qstar has been added ?